### ISSUE TITLE
Suggestion: Mojang login, players support memorizing information

### ISSUE THREAD

**Issue by BingYanchi** [GeyserMC/Floodgate#119]
It is recommended to give an option to remember when choosing Mojang to log in. No need to pass the verification on the Java side, just confirm that the Mojang account password can log in, and then bind the xbox id.

If possible, I would like to add another one to log in directly with a user name. Just enter the user name to log in. (Suitable for some servers with login plugins) I hope these two functions can be implemented at the same time, because there are some servers that can use Fastlogin to make Mojang logged-in players and offline logged-in players play together.

**Comment by Tim203** [comment]
I don't think that I understand your feature request. You don't have to login or enter any credentials when you use Floodgate

**Comment by BingYanchi** [comment]
> I don't think that I understand your feature request. You don't have to login or enter any credentials when you use Floodgate

It may be my problem. I am not sure if this feature belongs to Geyser or Floodgate. What I hope is that Geyser can record the login information when logging in and bind it.

**Comment by Tim203** [comment]
If you want something to be added to Geyser then you should make a feature request in Geyser. I'll transfer the issue to the Geyser repo
